boards/native/native_sim/doc/index.rst

```diff
@@ -510,44 +510,44 @@ The following peripherals are currently provided with this board:
 
 .. _native_ptty_uart:
 
-PTTY UART
+PTY UART
 =========
 
-This driver can be configured with :kconfig:option:`CONFIG_UART_NATIVE_POSIX`
-to instantiate up to two UARTs. By default only one UART is enabled.
-With :kconfig:option:`CONFIG_UART_NATIVE_POSIX_PORT_1_ENABLE`
-you can enable the second one.
-
-For the first UART, it can link it to a new
-pseudoterminal (i.e. :file:`/dev/pts{<nbr>}`), or map the UART input and
-output to the executable's ``stdin`` and ``stdout``.
-This is chosen by selecting either
-:kconfig:option:`CONFIG_NATIVE_UART_0_ON_OWN_PTY` or
-:kconfig:option:`CONFIG_NATIVE_UART_0_ON_STDINOUT`
-For interactive use with the :ref:`shell_api`, choose the first (OWN_PTY) option.
-The second (STDINOUT) option can be used with the shell for automated
-testing, such as when piping other processes' output to control it.
-This is because the shell subsystem expects access to a raw terminal,
+This driver is automatically enabled when devicetree contains nodes with the
+``"zephyr,native-pty-uart"`` compatible property and ``okay`` status and
+:kconfig:option:`CONFIG_SERIAL` is set.
+By default one ready UART of this type is setup in DTS, but any number can be enabled as desired.
+
+Normally these UARTs are connected to new pseudoterminals PTYs, i.e. :file:`/dev/pts{<nbr>}`,
+but it is also possible to map one of them to the executable's ``stdin`` and ``stdout``.
+This can be done in two ways, either with the command line option ``--<uart_name>_stdinout``
+(where ``<uart_name>`` is the UART DTS node name), or, for the first PTY UART instance by chosing
+:kconfig:option:`CONFIG_UART_NATIVE_PTY_0_ON_STDINOUT` instead of the default
+:kconfig:option:`CONFIG_UART_NATIVE_PTY_0_ON_OWN_PTY`.
+For interactive use with the :ref:`shell_api`, it is recommended to choose the PTY option.
+The ``STDINOUT`` option can be used for automated testing, such as when piping other processes'
+output to control it. This is because the shell subsystem expects access to a raw terminal,
 which (by default) a normal Linux terminal is not.
 
-When :kconfig:option:`CONFIG_NATIVE_UART_0_ON_OWN_PTY` is chosen, the name of the
-newly created UART pseudo-terminal will be displayed in the console.
-If you want to interact with it manually, you should attach a terminal emulator
-to it. This can be done, for example with the command:
+When a UART is connected to a new PTY, the name of the newly created UART pseudo-terminal will be
+displayed in the console.
+If you want to interact with it manually, you should attach a terminal emulator to it.
+This can be done, for example with the command:
```

Comment:
I wonder if it would be possible to also mention use of `gnome-terminal` here. The xterm is usually always found in the system so mentioning it here is fine, but xterm is somewhat annoying to use. And if one has Gnome, then gnome-terminal is more natural terminal to use.
I have added mentions of gnome-terminal to networking documentation, so I thought that it could be also mentioned here together with xterm in the native-sim doc too.
I am talking about this option
```
CONFIG_NATIVE_UART_AUTOATTACH_DEFAULT_CMD="gnome-terminal -- screen %s"
```

Reply:
Done.